content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,356 @@
+Rules
+-----
+
+Each structure has a set of *salary rules* to follow for accounting purposes. These rules are
+configured by the localization, and affect the *Accounting* application, so modifications to the
+default rules, or the creation of new rules, should only be done when necessary.
+
+To view all the rules, go to :menuselection:`Payroll app --> Configuration --> Rules`. Click on a
+structure (such as :guilabel:`Regular Pay`) to view all the rules.
+
+.. image:: configuration/rules.png
+   :align: center
+   :alt: Rules for each salary structure type.
+
+To make a new rule, click :guilabel:`Create`. A new rule form appears. Enter the information in the
+fields, then click :guilabel:`Save`.
+
+.. image:: configuration/new-rule.png
+   :align: center
+   :alt: Enter the information for the new rule.
+
+The required fields for a rule are:
+
+ - :guilabel:`Name`: Enter a name for the rule.
+ - :guilabel:`Category`: Select a category the rule applies to from the drop-down menu, or enter a
+   new one.
+ - :guilabel:`Code`: Enter a code to be used for this new rule. It is recommended to coordinate with
+   the accounting department for a code as this will affect them.
+ - :guilabel:`Salary Structure`: Select a salary structure the rule applies to from the drop-down
+   menu, or enter a new one.
+ - :guilabel:`Condition Based on`: In the :guilabel:`General` tab, select from the drop-down menu
+   whether the rule is :guilabel:`Always True` (always applies), a :guilabel:`Range` (applies to a
+   specific range, which is entered beneath the selection), or a :guilabel:`Python Expression` (the
+   code is entered beneath the selection).
+ - :guilabel:`Amount Type`: In the :guilabel:`General` tab, select from the drop-down menu whether
+   the amount is a :guilabel:`Fixed Amount`, a :guilabel:`Percentage (%)`, or a :guilabel:`Python
+   Code`. Depending on what is selected, the fixed amount, percentage, or Python code needs to be
+   entered next.
+
+Rule parameters
+---------------
+
+.. note::
+   Currently, the **Rule Parameters** feature found inside the :menuselection:`Payroll app -->
+   Configuration` menu is still in development and only serves a specific use case for Belgian
+   markets. The documentation will be updated when this section has matured to more markets.
+
+Other input types
+-----------------
+
+When creating payslips, it is sometimes necessary to add other entries for specific circumstances,
+like expenses, reimbursements, or deductions. These other inputs can be configured by going to
+:menuselection:`Payroll --> Configuration --> Other Input Types`.
+
+.. image:: configuration/other-input.png
+   :align: center
+   :alt: Other input types for payroll.
+
+To create a new :guilabel:`Input Type`, click the :guilabel:`Create` button. Enter the
+:guilabel:`Description`, the :guilabel:`Code`, and which structure it applies to. Click the
+:guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete the entry.
```

Comment:
No need for a gui on _input type_, and adding the **Availability in Structure** field for added context
```suggestion
To create a new input type, click the :guilabel:`Create` button. Enter the :guilabel:`Description`,
the :guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
field. Click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
the entry.
```